pkg/models/operational_solution.go

```diff
@@ -9,6 +9,7 @@ import (
 // OperationalSolution represents the specific implemented solution to an Operational Need
 type OperationalSolution struct {
 	baseStruct
+	operationalNeedRelation
 	OperationalNeedID uuid.UUID               `json:"operationalNeedID" db:"operational_need_id"`
```

Comment:
This should be removed, as the OperationalNeedID field comes from the embedded struct already, so this is redundant.

Reply:
When you run the code, one of the variables stays null unless you set it 
![image](https://user-images.githubusercontent.com/99294987/214339488-5905d330-4369-4ca9-93e9-87849759af9c.png)

Reply:
This is corrected removing the extraneous field
![image](https://user-images.githubusercontent.com/99294987/214342072-3127958c-fd97-44c8-87e2-a7032053a429.png)

Reply:
I resolved this